mlflow/tracing/export/mlflow_v3.py

```diff
@@ -87,27 +87,28 @@ def _log_trace(self, trace: Trace, prompts: Sequence[PromptVersion]):
         """
         try:
             if trace:
-                try:
-                    add_size_bytes_to_trace_metadata(trace)
-                except Exception:
-                    _logger.warning("Failed to add size bytes to trace metadata.", exc_info=True)
+                add_size_stats_to_trace_metadata(trace)
                 returned_trace_info = self._client.start_trace(trace.info)
                 self._client._upload_trace_data(returned_trace_info, trace.data)
-                # Always run prompt linking asynchronously since (1) prompt linking API calls
-                # would otherwise add latency to the export procedure and (2) prompt linking is not
-                # critical for trace export (if the prompt fails to link, the user's workflow is
-                # minorly affected), so we don't have to await successful linking
-                try_link_prompts_to_trace(
-                    client=self._client,
-                    trace_id=returned_trace_info.trace_id,
-                    prompts=prompts,
-                    synchronous=False,
-                )
             else:
                 _logger.warning("No trace or trace info provided, unable to export")
         except Exception as e:
             _logger.warning(f"Failed to send trace to MLflow backend: {e}")
 
+        try:
```

Comment:
Why do we move this out? returned_trace_info could be undefined if trace not existing